docs/sources/operators-guide/configuring/about-configurations.md

```diff
@@ -15,6 +15,36 @@ A given configuration loads at startup and cannot be modified at runtime. Howeve
 
 To see the current configuration state of any component, use the [`/config`]({{< relref "../reference-http-api/index.md#configuration" >}}) or [`/runtime_config`]({{< relref "../reference-http-api/index.md#runtime-configuration" >}}) HTTP API endpoint.
 
+## Common configurations
+
+Some configurations, like object storage backend, are repeated for multiple components.
+To avoid the repetition in the configuration file, the [`common`]({{< relref "../configuring/reference-configuration-parameters/index.md#common" >}}) configuration section or `-common.*` CLI flags can be used.
```

Comment:
```suggestion
To avoid the repetition in the configuration file, use the [`common`]({{< relref "../configuring/reference-configuration-parameters/index.md#common" >}}) configuration section or `-common.*` CLI flags.
```